### PR TITLE
Improve choropleth interaction-performance

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
This PR tries to improve the choropleth performance during interaction. It's a draft PR until it's tested across different browsers.

### CODE REVIEW

File: src/components/choropleth/Choropleth.tsx [hunk 1]
```diff
@@ -104,30 +103,19 @@ export function Choropleth<T>(props: TProps<T>) {
...
+        (tooltip: TooltipSettings | null) => set({ tooltip }),
+        120
+      ),
     }))
   );
 
   const clipPathId = useRef(`_${Math.random().toString(36).substring(2, 15)}`);
-  const timeout = useRef<any>(-1);
-  const isLargeScreen = useMediaQuery('(min-width: 1000px)');
+  const [isTouchDevice, setIsTouchDevice] = useState(false);
```
Comment (VWSCoronaDashboard10) [inline src/components/choropleth/Choropleth.tsx:118]: I assumed the mediaquery was in fact trying to target touch devices

Comment (VWSCoronaDashboard1) [reply src/components/choropleth/Choropleth.tsx:118]: Indeed it does

**Comment by VWSCoronaDashboard10** [comment]
Performance on safari isn't that good with these changes, back to the drawing board.

**Comment by VWSCoronaDashboard10** [comment]
Closed in favor of #786.

**Pull request closed without merging**